src/System.Private.CoreLib/src/Internal/Globalization/CultureDataSupport.cs

```diff
@@ -0,0 +1,21 @@
+// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+using System;
+using System.Globalization;
+
+namespace Internal.Globalization
+{
+    public static class CultureDataSupport
+    {
+        /// <summary>
+        /// Check whether CultureData exists for specified language
+        /// This API is used for WindowsRuntimeResourceManager in System.Runtime.WindowsRuntime
+        /// </summary>
+        public static bool IsCultureDataExists(string language)
```

Comment:
This argument is not language. I would call this `bool IsValidCulture(string cultureName)` and put into WindowsRuntimeResourceManagerBase so that all resource glue is together.